automation/src/test/scala/org/broadinstitute/dsde/test/api/orch/PublishedWorkspaceSpec.scala

```diff
@@ -48,7 +57,7 @@ class PublishedWorkspaceSpec extends FreeSpec with WorkspaceFixtures with Billin
               }
             }
           }
-        }
+        }(UserPool.chooseProjectOwner.makeAuthToken(billingScopes))
```

Comment:
just checking because I don't know -- there are no curators in the UserPool?

Reply:
There might be. Through testing in Rawls, project owners are all users of the billing account defined via `ServiceTestConfig`. I don't think the actual role matters tbh - it's just that the users are different.